src/cpu/aarch64/cpu_isa_traits.hpp

```diff
@@ -1,7 +1,7 @@
 /*******************************************************************************
 * Copyright 2018-2023 Intel Corporation
 * Copyright 2020-2024 FUJITSU LIMITED
-* Copyright 2023 Arm Ltd. and affiliates 
+* Copyright 2023, 2025 Arm Ltd. and affiliates 
```

Comment:
Maybe dash?

Reply:
Our convention is to use a dash only when all years in the range had a modification. In this case there was no change in 2024.